packages/api/src/APIProvider.tsx

```diff
@@ -1,13 +1,14 @@
 import React, { PropsWithChildren } from 'react';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+// import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
```

Comment:
can we remove this commented code

Reply:
@suisin-deriv i think this was intentional. please see this [link](https://deriv-group.slack.com/archives/C3AHMSM3P/p1685000609101639)